test/clojure/clojurewerkz/ogre/suite/has_test.clj

```diff
@@ -0,0 +1,94 @@
+(ns clojurewerkz.ogre.suite.has-test
+  (:require [clojure.test :refer [deftest testing is]]
+            [clojurewerkz.ogre.core :as q]
+            [clojurewerkz.ogre.vertex :as v]
+            [clojurewerkz.ogre.test-util :as u]))
+
+(defn get_g_V_outXcreatedX_hasXname__mapXlengthX_isXgtX3XXX_name [g]
```

Comment:
Please follow the code style ClojureWerkz projects have. Argument vector should be on the line after `defn`.